website/docs/sql-reference/sql-distinct.md

```diff
@@ -0,0 +1,60 @@
+---
+id: distinct
+title: SQL DISTINCT
+description: Using DISTINCT in a SELECT statement will force a query to only return non-duplicate rows. You may commonly see a DISTINCT clause in COUNT functions to get counts of distinct rows.
+slug: /sql-reference/distinct
+---
+
+<head>
+    <title>Working with the SQL DISTINCT</title>
+</head>
+
+Let’s just put it out there: at one point in your data work, you’ll encounter duplicates in your data. They may be introduced from a faulty data source (less likely) or created during the joining and transforming of data (more likely). You may need a more sophisticated or refactored solution for the latter scenario, but it never hurts to know how to use DISTINCT in a query.
```

Comment:
Not sure I agree w/ the less likely / more likely tags here!  They don't add a ton of clarity, think your point is stronger without them

Reply:
ok personally they make sense to me since I'm trying to suggest that duplicates in data are often likely self-induced (vs naturally there), but omitting it is also fine cc @johnblust